server/services/geocodes.js

```diff
@@ -0,0 +1,46 @@
+const { dbClient, collections } = require('../db');
+
+const getPointsFromPostalCodes = async (postalCodes) => {
+  const queryPoint = async (postalCode) => {
+    const spacedCode = `${postalCode.substring(0, 3)} ${postalCode.substring(3)}`;
+    const request = await dbClient.db[collections.GEOCODES].findOne({
+      postal_code: spacedCode,
+    });
+
+    if (request) {
+      // Happy path, full match on postal code
+      return {
+        lat: request.latitude,
+        lng: request.longitude,
+        match: request.postal_code.replace(' ', ''),
+      };
+    }
+    const approxRequest = await dbClient.db[collections.GEOCODES].findOne({
+      'postal_code like': `${spacedCode.slice(0, -1)}%`,
+    });
+
+    if (approxRequest) {
+      return {
+        lat: approxRequest.latitude,
+        lng: approxRequest.longitude,
+        match: approxRequest.postal_code.replace(' ', ''),
+      };
+    }
+
+    return { match: null };
+  };
+
+  if (Array.isArray(postalCodes)) {
```

Comment:
Hasn't been used yet, but I assume that once we have this running properly we will want to make several postal code lookups at one rather than one at a time.